crates/biome_cli/tests/cases/included_files.rs

```diff
@@ -92,6 +92,46 @@ fn does_not_handle_included_files_if_overridden_by_ignore() {
     ));
 }
 
+#[test]
+fn does_not_handle_files_in_ignored_folder() {
+    let mut console = BufferConsole::default();
+    let fs = MemoryFileSystem::default();
+    let file_path = Utf8Path::new("biome.json");
+    fs.insert(
+        file_path.into(),
+        r#"{
+  "files": { "includes": ["test.js", "!**/folder"] }
+}
+"#
+        .as_bytes(),
+    );
+
+    let test = Utf8Path::new("test.js");
+    fs.insert(test.into(), UNFORMATTED.as_bytes());
+
+    let test2 = Utf8Path::new("folder/test2.js");
+    fs.insert(test2.into(), UNFORMATTED.as_bytes());
```

Comment:
this test doesn’t actually test what you want it to, see https://github.com/biomejs/biome/issues/7268#issuecomment-3199814512

so this whole feature was shipped completely broken and is breaking user code as we speak.